.travis.yml

```diff
@@ -0,0 +1,15 @@
+sudo: false
+
+language: node_js
+node_js:
+  - "8"
+
+cache:
+  directories:
+    - node_modules
+
+before_script:
+  - npm i
+
+script:
+  - eslint .
```

Comment:
Does this really work? Can you add a "lint": "eslint .", script to package.json and add `npm run lint` to .travis.yml.

I think it would execute a globally installed eslint unless Travis CI is doing something extra magic.

Reply:
You're right, it should be `$(npm bin)/eslint` will update.